CONTRIBUTING.md

```diff
@@ -127,11 +127,11 @@ If you're happy to run tests only against an SQLite database, you can skip this
 
 If you have Docker installed, use any of the following commands to start fresh local databases of the dialect of your choice:
 
-- `yarn start-mariadb`
-- `yarn start-mysql`
-- `yarn start-postgres`
+- `yarn start-mariadb-oldest` (for MariaDB 10.3.0) or `yarn start-mariadb-latest` (for MariaDB 10.9.3)
+- `yarn start-mysql-oldest` (for MySQL 5.7.0) or `yarn start-mysql-latest` (for MySQL 8.0.30)
+- `yarn start-postgres-oldest` (for Postgres 10.21.1) or `yarn start-postgres-latest` (for Postgres 12.12.1)
```

Comment:
```suggestion
- `yarn start-postgres-oldest` (for Postgres 10) or `yarn start-postgres-latest` (for Postgres 12)
```
In this case even major versions since minor versions should not contain breaking changes.

Reply:
Fixed in latest commit